test/factree/pathfinder_test.rb

```diff
@@ -0,0 +1,60 @@
+require 'test_helper'
+
+describe Factree::Pathfinder do
+  describe "#find_node_sequence" do
+    let(:facts) do
```

Comment:
Yeah, I've seen that [comment](http://docs.seattlerb.org/minitest/Minitest/Spec/DSL.html#method-i-let) in the Minitest docs.

I have to admit that `let` is one pattern I like for testing. Memoized functions are really useful in that context.